### ISSUE TITLE
Clarify terminology: SmartProxy/SmartProxyServer, Internal/External/Isolated

### ISSUE THREAD

**Issue by aneta-petrova** [theforeman/foreman-documentation#2774]
I am unsure about "isolated" here. Maybe reword to "than an {SmartProxyServer}" instead?

IMHO we should drop "internal", "external", and possibly "isolated" and instead use "SmartProxy" and "SmartProxyServer" as attributes.

_Originally posted by @maximiliankolb in https://github.com/theforeman/foreman-documentation/pull/2607#discussion_r1420672311_

**Comment by asteflova** [comment]
Re: isolated/external/internal:

* We shouldn't use "isolated capsule" or "isolated host": "Capsule/CapsuleServer" or "host" is enough.
* Specifically for files guides/common/modules/con_smartproxy-networking.adoc, occurrences of "isolated" still persist in the diagrams, so these need to be updated.

Re: SmartProxy/SmartProxyServer: We need to clarify these terms and when to use them. @maximiliankolb @Lennonka

**Comment by maximiliankolb** [comment]
Closed https://github.com/theforeman/foreman-documentation/issues/2867

### Reword "internal Smart Proxy" and "external Smart Proxy" #2867 

* fix glossary
* architecture guide: explain difference between Smart Proxy and Smart Proxy Server
* possibly: fix images

**Comment by maximiliankolb** [comment]
"isolated" is not used anymore to describe Smart Proxies.
